sqlx-core/src/testing/mod.rs

```diff
@@ -259,3 +260,9 @@ async fn setup_test_db<DB: Database>(
         .await
         .expect("failed to close setup connection");
 }
+
+pub fn get_current_run_id() -> Option<String> {
+    // Currently we have to add a special case for any testrunners that spawns tests in separate processes
+    // The intention is to get a `CARGO_RUN_ID` merged into cargo and then use it instead in the future
```

Comment:
A link to an RFC or issue on the Cargo repo would be nice for additional context.

Reply:
I will remove this section, the cargo guys seem unlikely to add it